frontend/pages/queries/ManageQueriesPage/components/ManageAutomationsModal/ManageAutomationsModal.tsx

```diff
@@ -66,8 +66,14 @@ const ManageAutomationsModal = ({
   // TODO: Error handling, if any
   const [errors, setErrors] = useState<{ [key: string]: string }>({});
 
+  // Client side sort queries alphabetically
+  const sortedAvailableQueries =
```

Comment:
Oh ya! Future consideration for sure

Reply:
QQ: Are y'all planning to load test the changes in the UI with a bunch of queries? If so, it would be good to make sure the UI table and modal are called out in the test plan so any client-side search/sort functionality gets a solid workout 🏋️‍♀️💪